server/views/layouts/publicLayout.handlebars

```diff
@@ -24,18 +25,27 @@
   {{/if}}
 
   {{prefixScript '/static/vendor.js' 'head-scripts'}}
+  {{prefixModuleScript '/assets/js/index.js' 'head-scripts'}}
   {{prefixScript '/static/browserConfig' 'head-scripts'}}
-  {{prefixScript '/static/kth-style/js/menus.js' 'head-scripts'}}
   {{prefixScript '/static/kth-style/js/backtotop.js' 'head-scripts'}}
   {{{render 'head-scripts'}}}
 
   <script src="{{toolbarUrl}}"></script>
+  <script type="module">
+    import { MenuPanel } from '{{proxyPrefix}}/assets/js/index.js'
+    MenuPanel.init(document.querySelector('.kth-header'), document.querySelectorAll('.kth-mega-menu .kth-menu-item'))
```

Comment:
`{{{blocks.megaMenu}}}` adds at `<script>` that inits itself, so I don't this init isn't needed.

Reply:
Yep. Looks like you are correct. ⭐  Fixed it.